docs/reference/api/drivers/UARTSerial.md

```diff
@@ -0,0 +1,54 @@
+## UARTSerial
+
+The UARTSerial class provides an alternative and enhanced version of the [Serial](https://os.mbed.com/docs/v5.6/reference/serial.html) interface. 
+
+
+Features:
+
+* It uses the latest FileHandle API that permits non-blocking and device operations. It is now possible to use a serial device with the C library file handling functions.
```

Comment:
latest wont be true forever so can drop that, in fact, this was always possible with Stream so not sure why calling it out here.

Reply:
From what Kevin was saying, it was designed to replace the need for stream in this context....